docker-compose.yml

```diff
@@ -49,6 +50,20 @@ services:
             - "8070:8070"
         networks:
             - grobid
+        healthcheck:
+            test: ["CMD", "bash", "-c", "echo > /dev/tcp/localhost/8070"]
+            interval: 10s
+            timeout: 10s
+            retries: 5
+    grobid-warmup:
+        image: willwill/wait-for-it
+        command: grobid:8070 -- /bin/sh -c "curl -v --form input=@/tmp/warmup-sample.pdf --form consolidateHeader=0 --form consolidateCitations=0 -o /dev/null http://grobid:8070/api/processFulltextDocument"
```

Comment:
`POST` is inferred automatically here. I can't know if the HTTP request is what Grobid expects but it seems similar to what I've seen before.
This container will execute and then exit. I think it needs a [`restart: on-failure`](https://docs.docker.com/compose/compose-file/) option so that it stays stopped when exiting with 0. Not sure what will happen on a reboot, the desired behavior is to execute it again.